src/main/java/nearsoft/skytouch/task/model/Product.java

```diff
@@ -0,0 +1,43 @@
+package nearsoft.skytouch.task.model;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class Product {
+    @NotEmpty
+    @NotNull
+    private String name;
+    @NotEmpty
+    @NotNull
+    private String description;
+    @NotEmpty
+    @NotNull
+    private Long price;
```

Comment:
you should be using a floating point data type for prices

Reply:
I'm using Long only for ease in this case

Reply:
It is not bad idea to use integer data types for prices with the meaning of cents... using floating point data types it is possible to loose cents during calculations